docs/writing-docs/doc-blocks.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Doc blocks
+---
+
+Storybook offers a number of doc blocks to help document your components and other aspects of your project.
+
+There are two common ways to use doc blocks in Storybook, within MDX and as part of the docs page template.
+
+## Within MDX
+
+The blocks are most commonly used within Storybook's [MDX documentation](./mdx.md):
+
+![Screenshot of mdx content](TK)
+
+<!-- prettier-ignore-start -->
+```md
+{/* ButtonDocs.mdx */}
+
+import { Meta, Primary, Controls, Story } from '@storybook/blocks';
+import * as ButtonStories from './Button.stories';
+
+<Meta of={ButtonStories} />
+
+# Button
+
+A button is ...
+
+<Primary />
+
+## Props
+
+<Controls />
+
+## Stories
+
+### Primary
+
+Button can be of primary importance
+
+<Story of={ButtonStories.Primary} />
+
+Button can be of secondary importance
+
+<Story of={ButtonStories.Secondary} />
+
+{/* ... */}
+```
+<!-- prettier-ignore-end -->
+
+## Customizing the automatic docs page
+
+The blocks are also used to define the page template for [automatics docs](./autodocs.md). For example, here's the default template:
+
+![Screenshot of automatic docs template](TK)
+
+```jsx
+import { Title, Subtitle, Description, Primary, Controls, Stories } from '@storybook/blocks';
+
+export const autoDocsTemplate = () => (
+  <>
+    <Title />
+    <Subtitle />
+    <Description />
+    <Primary />
+    <Controls />
+    <Stories />
+  </>
+);
```

Comment:
unfortunately this default template is being changed - and made more complex - in https://github.com/storybookjs/storybook/pull/21178
See https://github.com/storybookjs/storybook/pull/21178#issuecomment-1439164344

I'm not sure what to do here, it's a shame that it is now not as straight forward and contains actually logic. we could just update this to match, or reference the file directly? Up to you.